### PR TITLE
Added both starter methods

### PR DESCRIPTION
**Pull request by Spilare**
Added 2 methods,
- Starter (Soft resets for Emerald starters based on the option in config)
- FRLGStarter (Soft resets for FRLG starters based on which one you stand in front and save)

note all these methods must be started from the press start screen after saving in front of the bag/ball

### CODE REVIEW

**Comment by Spilare** [comment]
Added pid file to keep track and made it all neat with one method
note that for FRLG starters as it is not required to change the name it may get confused if the user doesnt update the starter tag
it wont affect much except speed if they swap hunt

File: modules/Memory.py [hunk 1]
```diff
@@ -126,7 +126,8 @@ def __init__(self, pid):
                 os._exit(1)
         time.sleep(0.5)
 
-
+def GetGame():
+    return mGBA.game
```
Comment (40Cakes) [inline modules/Memory.py:130]: I don't understand the point of this, why don't you just grab this directly from the mGBA class?

Comment (Spilare) [reply modules/Memory.py:130]: from where I needed to access it I would need to import some stuff, honestly I wasnt sure if you could import a class or not so I did this so I could just import the required info

File: modules/Memory.py [hunk 2]
```diff
@@ -502,7 +503,7 @@ def ParsePokemon(b_Pokemon: bytes):
 def GetParty():
     party = {}
     b_gPlayerParty = ReadSymbol('gPlayerParty')
-    party_count = int.from_bytes(ReadSymbol('gPlayerPartyCount'))
+    party_count = int.from_bytes(ReadSymbol('gPlayerPartyCount'),'little')
```
[the remark this answers is not in the document]
Comment (Spilare) [reply modules/Memory.py:506]: I tried it but it wasnt working not sure why tho

**Pull request merged**